canvas_modules/common-canvas/src/common-properties/controls/expression/expression-builder/expression-select-field-function.jsx

```diff
@@ -608,7 +608,8 @@ export default class ExpressionSelectFieldOrFunction extends React.Component {
 				<Layer level={this.props.controller.getLight() ? 1 : 0}>
 					<Dropdown
 						id={"properties-expression-function-select-dropdown-" + this.uuid}
-						label={label}
+						label={selectedFunctionItem}
+						selectedItem={selectedFunctionItem}
```

Comment:
Does this work with other languages?  I don't think it would since items list val and label would be different.

Reply:
I have added labels using existing text as Fields and Functions. In other parts of code where `Dropdown` I can see generic text like [here](https://github.com/elyra-ai/canvas/blob/19c1d42bf830b88c4e88c907ebea9a8a45f7da53/canvas_modules/harness/src/client/components/common-properties-conditions.jsx#L208)

Reply:
Isn't this still using the label (`locLabel`) for the selctedItem even though the selectItem would be using the `value` to determine selection?

Reply:
Looks like it label is being used.  Seems like a carbon bug but we can keep it.